db_import/socrata_functions.R

```diff
@@ -59,8 +91,9 @@ create_sections_tables = function(
   # The data come in with many "NULL" entries, but they are literally interpreted as
   # a string reading "NULL" instead of NA's
   
-  # Replace 'NULL' with NA's
-  lapply( names(sections_tbl), function(x) { sections_tbl[get(x) == "NULL", (x) := NA] } )
+  # Replace 'NULL' with NA's 
```

Comment:
This can probably be reverted as well.  Are the columns at this point all character or have they been converted to to their datatypes at this point?

Reply:
Aside from the `begindate` as noted above, all columns are characters and have yet to be cast to their expected final datatypes. That casting happens in the `write_to_stage` function which is called immediately after this function (`create_sections_tables`) in the import script.

Reply:
@landisrm I've commit some minor clean-up. 

The function `read.socrata` now automatically casts dates as POSIXct types so the `create_sections_tables` function now casts all columns to character and leave type handling to `write_to_stage`. Spot checking the dates, it seems this conversion isn't causing issues.